tests/src/test/java/io/streamnative/pulsar/handlers/kop/DistributedClusterTest.java

```diff
@@ -495,4 +495,59 @@ public void testOneBrokerShutdown() throws Exception {
         kafkaConsumeCommitMessage(kConsumer1, totalMsgs, messageStrPrefix, topicPartitions);
         kafkaConsumeCommitMessage(kConsumer2, totalMsgs, messageStrPrefix, topicPartitions);
     }
+
+    @Test(timeOut = 30000)
+    public void testPollEmptyTopic() throws Exception {
```

Comment:
Could you move it to another test class because `DistributedClusterTest` is disabled now? And it's not related to multiple brokers.

Reply:
OK, I move to another test class.